hack/tests/e2e-create-prod.sh

```diff
@@ -1,23 +1,8 @@
 #!/bin/bash -e
 
-cleanup() {
-    set +e
-
-    if [[ -n "$ARTIFACTS" ]]; then
-        exec &>"$ARTIFACTS/cleanup"
-    fi
-
-    make artifacts
-
-    if [[ -n "$NO_DELETE" ]]; then
-        return
-    fi
-    make delete
-    az group delete -g "$RESOURCEGROUP" --yes --no-wait
-}
 trap cleanup EXIT
```

Comment:
can the trap be moved to ci-prepare.sh?

Reply:
that would mean that `cleanup()` gets called at the end of every script, which may not be something you want to do, I rather keep it on each script to maintain some control